Can't build on homebrew CI

There is an uninformative error when builds on Brew Jankins farm.
```
tar xf /Library/Caches/Homebrew/git-lfs-0.5.4.tar.gz
==> ./script/bootstrap
panic: exit status 128

goroutine 1 [running]:
main.mainBuild()
	/private/tmp/git-lfs20150731-31872-fyr904/git-lfs-0.5.4/script/build.go:40 +0x249
main.main()
	/private/tmp/git-lfs20150731-31872-fyr904/git-lfs-0.5.4/script/script.go:19 +0x8c
exit status 2
```
http://bot.brew.sh/job/Homebrew%20Pull%20Requests/30547/version=mavericks/testReport/junit/brew-test-bot/mavericks/install_git_lfs/

I'm not Go developer, so can't figured what happens.

--- a/script/build.go
+++ b/script/build.go
@@ -36,9 +36,6 @@ func mainBuild() {
 	}
 
 	cmd, err := exec.Command("git", "rev-parse", "--short", "HEAD").Output()
-	if err != nil {
-		panic(err)
-	}
 
 	if len(cmd) > 0 {
 		LdFlag = strings.TrimSpace("-X github.com/github/git-lfs/lfs.GitCommit " + string(cmd))

--- a/script/build.go
+++ b/script/build.go
@@ -35,7 +35,7 @@ func mainBuild() {
 		return
 	}
 
-	cmd, err := exec.Command("git", "rev-parse", "--short", "HEAD").Output()
+	cmd, _ := exec.Command("git", "rev-parse", "--short", "HEAD").Output()
 
 	if len(cmd) > 0 {
 		LdFlag = strings.TrimSpace("-X github.com/github/git-lfs/lfs.GitCommit " + string(cmd))

